pyof/v0x01/foundation/base.py

```diff
@@ -111,18 +111,18 @@ def value(self):
             return self._value
 
     def pack(self, value=None):
-        """Pack the value as a binary representation.
+        r"""Pack the value as a binary representation.
```

Comment:
It means *raw string*. The difference is that backslashes are treated like a regular character. That's why b'\x01' was not being interpreted correctly.